doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -138,7 +138,8 @@ described elsewhere
    Searched objects can be filtered by patterns, by the constants they
    contain (identified by their name or a notation) and by their
    names.
```

Comment:
```suggestion
   contain (identified by their name or a notation), by their
   names and by their location (e.g. :n:'head`).
```

This sentence should appear right after the "This command can be used ..." sentence.  Would you move it?

BTW, the formatting of the text describing Search is not standard, but not something to change in this PR.